### PR TITLE
[wix-style-react] - <Calendar /> - Add aria label to weekdayElement

### PR DESCRIPTION
**Pull request by bamai**
https://jira.wixpress.com/browse/DSM-1441

### CODE REVIEW

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6633

File: packages/wix-style-react/src/Calendar/Calendar.js [hunk 1]
```diff
@@ -267,6 +267,21 @@ export default class Calendar extends React.PureComponent {
...
+  _createWeekdayElement = localeUtils => {
+    return ({ weekday }) => {
+      const weekdayShort = localeUtils.formatWeekdayShort(weekday);
+      const weekdayLong = localeUtils.formatWeekdayLong(weekday);
+      return (
+        <div
+          className={st('DayPicker-Weekday', classes.weekday)}
+          aria-label={weekdayLong}
+        >
+          <abbr>{weekdayShort}</abbr>
```
Comment (sivanharel) [inline packages/wix-style-react/src/Calendar/Calendar.js:279]:
```suggestion
          <abbr title={weekdayLong}>{weekdayShort}</abbr>
```

File: packages/wix-style-react/src/Calendar/Calendar.js [hunk 2]
```diff
@@ -267,6 +267,21 @@ export default class Calendar extends React.PureComponent {
...
   };
 
+  _createWeekdayElement = localeUtils => {
+    return ({ weekday }) => {
+      const weekdayShort = localeUtils.formatWeekdayShort(weekday);
+      const weekdayLong = localeUtils.formatWeekdayLong(weekday);
+      return (
+        <div
+          className={st('DayPicker-Weekday', classes.weekday)}
+          aria-label={weekdayLong}
```
Comment (sivanharel) [inline packages/wix-style-react/src/Calendar/Calendar.js:277]: We should add:
`role="columnheader"`.
same as they added in the original element

**Review by sivanharel — COMMENTED** [review]
Very nice!!!!

**Review by sivanharel — APPROVED** [review]
Great job:)

**Pull request merged**